ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Solaris.yml

```diff
@@ -123,6 +190,36 @@
     - build_tools
     - cups
 
+- name: Check if cups-1.5.4-source is present (sparcv9)
```

Comment:
We seem to download, build and install 1.4.4 above. Why do we need both versions? If it's required, please add an explanatory comment for the future.

Reply:
done